baseapp_chats/graphql/mutations.py

```
@@ -404,6 +384,164 @@ def mutate_and_get_payload(
+        )
+
+    @classmethod
+    @login_required
+    def mutate_and_get_payload(
+        cls,
+        root,
+        info: graphene.ResolveInfo,
+        profile_id: str,
+        participant_profile_id: str,
+        room_ids: list[str],
+        **input,
```

Ruff (0.15.21)

[error] Function argument `input` is shadowing a Python builtin

(A002)